### PR TITLE
Choose .list as default route in config menu

### LINKED ISSUE #1610: Some navigation tabs link to empty sites
Some navigation tabs link to empty sites if you're already on them.

Example: Click on Audit and then click again on audit. You will see a "empty" site.

Following tabs are affected:

Components ( `/#/component/clienttype` and `/#/component/` )
Audit ( `/#/audit/log` and `/#/audit/` )

Config > Policies ( `/#/config/policies/list` and `/#/config/policies/` )
Config > Events ( `/#/config/events/list` and `/#/config/events/` )
Config > Periodic Tasks ( `/#/config/periodictasks/list` and` /#/config/periodictasks/` )
Config > Machines ( `/#/config/machineresolvers/list` and `/#/config/machineresolvers/` )
Config > Users ( `/#/config/resolvers/list` and `/#/config/resolvers/` )
Config > CAs ( `/#/config/caconnectors/list` and `/#/config/caconnectors/` )

The tabs Tokens, Users/User, Machines, Config, Config > System, Config > Tokens and Config > Realms are not affected.

### PR DESCRIPTION
**Pull request by cornelinux**
Closes #1610

### CODE REVIEW

**Comment by cornelinux** [comment]
I am choosing @fredreichbier as technical coder and @tamaro-skaljic as issue creater for review.
@tamaro-skaljic you can checkout this PR and test run it with the UI.

File: privacyidea/static/components/config/states/states.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ angular.module('privacyideaApp.configStates', ['ui.router']).config(
                     templateUrl: configpath + "config.system.html"
                 })
                 .state('config.policies', {
-                    url: "/policies",
+                    url: "/policies/",
```
Comment (fredreichbier) [inline privacyidea/static/components/config/states/states.js]: Hm, on my local development server, I end up with a weird URL for the policies list: http://localhost:5000/#/config/policies//list (with a double slash). Maybe this is related to the change above?

Comment (cornelinux) [reply privacyidea/static/components/config/states/states.js]: I most probably is!
Where do you get this URL? (What was your "click way")

Comment (cornelinux) [reply privacyidea/static/components/config/states/states.js]: done in da05d0b7

**Comment by cornelinux** [comment]
@fredreichbier if this is fine for you now, I would like you to merge this, to avoid having this dangling till the end of the week.

**Comment by fredreichbier** [comment]
OK, I've checked that this PR fixes all the links listed by @tamaro-skaljic in #1610. So I'll merge this.

**Pull request merged**